src/scss/coral-talk-iframe/main.scss

```diff
@@ -408,6 +432,52 @@ body {
 	}
 }
 
+// reply indent
+
+// Unfortunately at the moment it looks like there is a bug in Corals code
+// I believe there should be a .coral-indent class we could use here
+// If this is fixed in the future we can reduce this to a single selector
+.coral-comment .coral-indent-1,
+.coral-comment .coral-indent-2,
+.coral-comment .coral-indent-3,
+.coral-comment .coral-indent-4,
+.coral-comment .coral-indent-5,
+.coral-comment .coral-indent-6 {
+	border-left-style: dotted;
+	border-width: 2px;
+	border-color: #807973;
+	border-image: url(https://www.ft.com/__origami/service/image/v2/images/raw/https%3A%2F%2Fs3-eu-west-1.amazonaws.com%2Fft-next-assets-prod%2Fassets%2Fcomments%2Fborder-dots.svg?source=next&format=svg);
```

Comment:
@notlee I had forgotten I was working in an `o-` component when doing this and reverted back to my next ways.

How do Origami normally handle image assets? Should I be adding this image somewhere thats not a random next S3 bucket?

Reply:
Ooops I totally skimmed past this, thanks for flagging! How come its needed given `border-width` and `border-left-style: dotted;`?

Reply:
Sorry this is my bad I should have removed `border-left-style` and also `border-color` as these were my original fallbacks that are no longer needed. However `border-width` is needed because its set on the global styles and would increase the width of the border and the image if its not set. I could probably make it specific to the left border though seeing as thats all we can about.

Reply:
Any thoughts on how I should do the image?

Reply:
Soo I'm not sure... none of our current components have any image assets that aren't in an imageset. Looks like the way we used to handle one off assets was to use [o-assets](https://github.com/Financial-Times/o-assets). The Build Service then serves the asset through the files endpoint `https://www.ft.com/__origami/service/build/v2/files/...` but I don't know how it would work for manual build users... @Financial-Times/origami-core?

The border looks like little circles, how come we can't achieve that with CSS? The spacing between the circles maybe?

Reply:
Seems like o-assets relies on the running app either having `bower_components`, or having copied the assets somewhere for each component and setting a variable. Yuk

Reply:
Unfortunately I couldn't find a css only solution for doing rounded dots at a small size and having control of the spacing :(

I would have normally pushed for just using `border-left: 2px dotted #807973` and ignored the spacing issue but unfortunately many browsers render squares instead of dots below a certain size, in chrome for instance this size is 4px.

Example -> https://codepen.io/GlynnPhillips/details/bGVLJLe

Reply:
happy to have the assert in the s3 bucket I think 👍

Reply:
agree with Jake for now :) opened an issue to figure out how we manage assets going forward https://github.com/Financial-Times/origami/issues/54